modules/wsl-distro.nix

```diff
@@ -10,6 +10,14 @@ in
   options.wsl = with types; {
     enable = mkEnableOption "support for running NixOS as a WSL distribution";
     useWindowsDriver = mkEnableOption "OpenGL driver from the Windows host";
+    kernelModules = mkOption {
+      type = listOf str;
+      default = [ ];
+      description = ''
+        The set of kernel modules to be loaded in the second stage of
+        the boot process via systemd-modules-load.service.
+      '';
+    };
```

Comment:
Why can't we reuse `boot.kernelModules` here?

Reply:
Since it is gated behind boot.kernel.enable which we don't want to enable.

Another option is to have something like a boolean wsl.loadKernelModules. That will write the file with the value based on boot.kernelModules. What do you prefer?